schema/swagger.yml

```diff
@@ -129,16 +143,19 @@ definitions:
     properties:
       epoch:
         type: string
+        format: date-time
         example: '2016-05-24T17:00:00.000Z'
       milestone:
-        type: string
+        type: integer
         example: 1081560729258
       build:
         type: string
         example: "v09:54:35 12/04/2017\n"
       commit:
         type: string
         example: 7199d4b67c3575d5f99d1c29436a02977eeb01a7
+        minLength: 40
+        maxLength: 40
```

Comment:
is this hash always 40 char from github?

Reply:
Yes it was a test case that I removed and added this validation check.